Directory.Build.props

```diff
@@ -1,10 +1,14 @@
 <Project>
   <PropertyGroup>
     <!--This should be passed from the VSTS build-->
-    <ClientSemVer Condition="'$(ClientSemVer)' == ''">1.0.0-localbuild</ClientSemVer>
+    <!-- This needs to be greater than or equal to the validation baseline version -->
+    <ClientSemVer Condition="'$(ClientSemVer)' == ''">2.7.0-localbuild</ClientSemVer>
```

Comment:
why 2.7.0?

Reply:
2.7 had breaking changes that we missed: https://github.com/AzureAD/microsoft-identity-web/compare/2.6.1...2.7.0#diff-f01bfd02584425a5c3f60da34d8e5b03cb533ec0689ae8a7f17e0b27e8b462d9

So I went to the oldest that didn't have breaking changes.